### ISSUE TITLE
Installation error: Computed hash did not match

### ISSUE THREAD

**Issue by srdamani** [Evidlo/remarkable_entware#26]
When installing on RM2 with version 3.13.1.1, I am getting this error:

```
root@reMarkable:~# wget -O - http://raw.githubusercontent.com/Evidlo/remarkable_entware/master/install.sh | sh
Connecting to raw.githubusercontent.com (185.199.111.133:80)
Connecting to raw.githubusercontent.com (185.199.109.133:443)
wget: note: TLS certificate validation not implemented
writing to stdout
Info: Checking for prerequisites and creating folders...
-                    100% |******************************************************************************************************************************************|  3736  0:00:00 ETA
written to stdout
Info: Opkg package manager deployment...
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/bin/opkg'
opkg                 100% |******************************************************************************************************************************************|  734k  0:00:00 ETA
'/opt/bin/opkg' saved
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/etc/opkg.conf'
opkg.conf            100% |******************************************************************************************************************************************|   171  0:00:00 ETA
'/opt/etc/opkg.conf' saved
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/lib/ld-2.27.so'
ld-2.27.so           100% |******************************************************************************************************************************************|  131k  0:00:00 ETA
'/opt/lib/ld-2.27.so' saved
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/lib/libc-2.27.so'
libc-2.27.so         100% |******************************************************************************************************************************************| 1218k  0:00:00 ETA
'/opt/lib/libc-2.27.so' saved
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/lib/libgcc_s.so.1'
libgcc_s.so.1        100% |******************************************************************************************************************************************| 46752  0:00:00 ETA
'/opt/lib/libgcc_s.so.1' saved
Connecting to bin.entware.net (158.160.45.54:80)
saving to '/opt/lib/libpthread-2.27.so'
libpthread-2.27.so   100% |******************************************************************************************************************************************| 92656  0:00:00 ETA
'/opt/lib/libpthread-2.27.so' saved
cat: read error: Is a directory
Computed hash did not match.
root@reMarkable:~# 
```

**Comment by axelmierczuk** [comment]
Getting this issue as well. There also seems to be an issue with (what seems to be) pre-emptive creation of the `/opt/lib/opkg` folder. I moved `mkdir -p /opt/lib/opkg` to after the hash check, and changed `lib/opkg` to `lib` on line 73 I am getting the following hash: `6b0e8c94b4c3549756c3ae9e144ac1fb`. Looks like the latest change to https://bin.entware.net/armv7sf-k3.2/installer/ was `24-Feb-2024`, which would explain the mis-match in hash since the last commit here was Aug 9, 2023.

**Comment by Evidlo** [comment]
Fixed with #27
